lib/controllers/state_controller.js

```diff
@@ -23,6 +23,20 @@ Bounce.prototype.set = function (key, value, callback) {
     return this;
   }).nodeify(callback);
 };
+Bounce.prototype.setDisplayProperty = function(name, value, callback) {
+  var existingProperty = _.where(this._bounceToken.displayProperties, {name: name});
```

Comment:
also just a thought but do we want these to be case sensitive? it is as it stands. is "DisplayName" different to "displayName" or "Displayname" possibly match based on lower case version of the name? and then if it exists delete it and reset it with the new name?